src/styles/getReportActionContextMenuStyles.js

```diff
@@ -0,0 +1,63 @@
+import CONST from '../CONST';
+import styles from './styles';
+import variables from './variables';
+import themeColors from './themes/default';
+
+/**
+ * Generate styles for the buttons in the mini comment actions menu.
+ *
+ * @param {String} [buttonState] - One of {'default', 'hovered', 'pressed'}
+ * @returns {Array}
+ */
+function getMiniButtonStyle(buttonState = CONST.BUTTON_STATES.DEFAULT) {
+    const defaultStyles = [styles.p1, styles.mv1, styles.mh2, {borderRadius: variables.componentBorderRadius}];
+    switch (buttonState) {
+        case CONST.BUTTON_STATES.HOVERED:
+            return [
+                ...defaultStyles,
+                {
+                    backgroundColor: themeColors.activeComponentBG,
```

Comment:
Just wanted to clarify here - the actual color value here should be `gray1` - currently I think it is actually using `gray2`. Let's use `gray1` for hover, and `gray2` for pressed.